zulip_bots/zulip_bots/bots/yoda/test_yoda.py

```diff
@@ -50,14 +50,14 @@ def test_bot(self):
 
         # Test help.
         bot_response = '''
-            This bot allows users to translate a sentence into
+            This bot will allow users to translate a sentence into
```

Comment:
It's updating the test for the yoda bot, so `tools/test-bots` doesn't fail on yoda.